assets/app/index.html

```diff
@@ -71,104 +118,60 @@
 
     <script src="config.js"></script>
```

Comment:
To be honest if that ordering matters I'd recommend doing it as a pre-bootstrap task, for example -> https://github.com/fabric8io/fabric8-console/blob/master/plugins/main/ts/mainPlugin.ts#L124-137